src/utils/helper-functions/typecheck.ts

```diff
@@ -1,10 +1,7 @@
 import { TSESTree } from '@typescript-eslint/experimental-utils'
 import { couldBeType } from 'tsutils-etc'
 
-function getParserServices(
-  // RuleContext<string, readonly unknown[]>>
-  context: Readonly<any>,
-) {
+function getParserServices(context: Readonly<any>) {
```

Comment:
I guess it's violating the `@typescript-eslint/no-explicit-any` rule, no? You could use this instead:

```suggestion
function getParserServices(
  context: TSESLint.RuleContext<string, readonly unknown[]>,
) {
```

Reply:
I was looking for these types, but couldn't find them, only in the dist folder.
Thanks!
I'll also use this to grab the store name from the settings, instead of passing the settings down to `readNgRxStoreNameFromSettings`

Reply:
Happened to me also with `RuleFixer` and I even opened an [issue](https://github.com/typescript-eslint/typescript-eslint/issues/2931) there. Since that discovery, I've been using it :)